### PR TITLE
chore: reimplement our comboboxes

### PR DESCRIPTION
**Pull request by raastinn**
## Changes
- Used `cmdk` command components use in `shadcn`
- Updated `Popover` component
- Deleted unused combobox components/props after refactor

## Pre-Review Checklist
* [x] I have considered backwards and forwards compatibility.
* [x] All changes are tested.
* [x] All touched code documentation is updated.
* [x] All tests pass in my local environment
* [x] Deepsource issues have been reviewed and addressed.
* [x] Commented code and `console` usages have been removed.
 
## Screenshots
_Only required for visual changes_

Screenshot from 2024-08-30 03-16-57
Screenshot from 2024-08-30 03-17-05

### CODE REVIEW

File: src/js/app/theme.ts [hunk 1]
```diff
@@ -82,7 +82,7 @@ export const theme: DefaultTheme = {
         yellowDarkest: "#B79A18",
     },
     fontFamily: {
-        monospace: "ui-monospace,SFMono-Regular,SF Mono,Menlo,Consolas,Liberation Mono,monospace",
+        monospace: "command-monospace,SFMono-Regular,SF Mono,Menlo,Consolas,Liberation Mono,monospace",
```
Comment (ReeceHoffmann) [inline src/js/app/theme.ts]: ?

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -29,7 +29,7 @@ jobs:
         id: meta
         uses: docker/metadata-action@98669ae865ea3cffbcbaa878cf57c20bbf1c6c38
         with:
-          images: ${{ env.REGISTRY }}/virtool/ui
+          images: ${{ env.REGISTRY }}/virtool/command
```
Comment (ReeceHoffmann) [inline .github/workflows/publish.yml]: Is this intended?

File: package.json [hunk 3]
```diff
@@ -21,6 +21,7 @@
         "@types/superagent": "^4.1.16",
         "class-variance-authority": "^0.7.0",
         "clsx": "^2.1.1",
+        "cmdk": "^1.0.0",
```
Comment (ReeceHoffmann) [inline package.json:24]: why this package over shadcn? Last update was 6 months ago. Might still be a good option though.

Also I think removing the old combobox allows us tor remove a dep.

File: vite.config.js [hunk 4]
```diff
@@ -51,7 +51,7 @@ export default defineConfig({
         }),
         sentryVitePlugin({
             org: "cfia-virtool",
-            project: "cloud-ui",
+            project: "cloud-command",
```
Comment (ReeceHoffmann) [inline vite.config.js]: ?

File: package.json (hunk 3 above)
Comment (raastinn) [reply package.json:24]: The way `shadcn` did it is using `Popover` and `cmdk`. There wasn't an actual package for it. Also removed `downshift` as a result of removing the old combobox

**Pull request closed without merging**